### PR TITLE
Updates to the Split Horizon EDS example

### PR DESCRIPTION
**Pull request by ymesika**
Updates to the doc after verifying the example with latest v1.1 build

### CODE REVIEW

**Comment by linsun** [comment]
@ymesika any particular reason why MCP needs to be disabled for this scenario?  I asked because that is not obvious and it is enabled by default.

**Comment by linsun** [comment]
cc @rvennam

**Comment by ymesika** [comment]
@linsun I'm not sure the instruction I have in this example fits the scenario where Pilot fetches the configuration from MCP and MCP is watching the remote k8s api. This still needs to be tested and examined and with the `useMCP=false` flag we choose the classical way of Pilot watching k8s.

I think it has no affect on `istio-remote` though so I will remove it.

File: content/docs/examples/multicluster/split-horizon-eds/index.md [hunk 1]
```diff
@@ -92,7 +93,7 @@ This will be used to access the `local` pilot securely using the ingress gateway
     {{< text bash >}}
     $ kubectl create --context=$CTX_LOCAL ns istio-system
     $ kubectl create --context=$CTX_LOCAL secret generic cacerts -n istio-system --from-file=samples/certs/ca-cert.pem --from-file=samples/certs/ca-key.pem --from-file=samples/certs/root-cert.pem --from-file=samples/certs/cert-chain.pem
-    $ kubectl apply --context=$CTX_LOCAL -f install/kubernetes/helm/istio/templates/crds.yaml
+    $ helm install install/kubernetes/helm/istio-init --name istio-init
```
Comment (frankbu) [inline content/docs/examples/multicluster/split-horizon-eds/index.md]: Which context is `helm install` working in? This command implies some required setup that was not mentioned in the earlier instructions - e.g., tiller should be running in the local cluster

**Comment by frankbu** [comment]
@linsun, `/lgtm` on doc PRs cause auto-merge so please don't `/lgtm` unless it's has been reviewed and you really want to trigger the merge.

File: content/docs/examples/multicluster/split-horizon-eds/index.md (hunk 1 above)
Comment (ymesika) [reply content/docs/examples/multicluster/split-horizon-eds/index.md]: Right. How do we install CRDs without Helm? Latest releases don't have a `crds.yaml` in them anymore.

Comment (frankbu) [reply content/docs/examples/multicluster/split-horizon-eds/index.md]: Good question. I think we do need a non-helm way to install the CRDs.

@andraxylia Do you know what the non-helm plan is?

Comment (linsun) [reply content/docs/examples/multicluster/split-horizon-eds/index.md]: You could do it via a loop to install CRDs one by one.  See https://preliminary.istio.io/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template

However using istio-init is also valid which is why it was created in the first place.

**Comment by linsun** [comment]
@frankbu I did review it, I missed the good context question you raised.  @ymesika can help address your concern in a separate WI.

**Pull request merged**